### ISSUE TITLE
Image example breaks/freezes tab

### ISSUE THREAD

**Issue by osanseviero** [gradio-app/gradio#1533]
### Describe the bug

I select an example image in the space, it does not fully load and then my Spaces page freezes. The space works well when uploading my own image.

https://huggingface.co/spaces/keras-io/EDSR

Even if the error is in user side (I don't know), the space should likely not freeze like this.

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Reproduction

Go to https://huggingface.co/spaces/keras-io/EDSR and click example image

### Screenshot

<img width="1201" alt="Screen Shot 2022-06-10 at 4 35 05 PM" src="https://user-images.githubusercontent.com/7246357/173088896-e24cefb9-b853-418c-902a-4691b4fe62d7.png">

### Logs

```shell
Traceback (most recent call last):
  File "/home/user/.local/lib/python3.8/site-packages/gradio/routes.py", line 255, in run_predict
    output = await app.blocks.process_api(
  File "/home/user/.local/lib/python3.8/site-packages/gradio/blocks.py", line 538, in process_api
    predictions, duration = await self.call_function(fn_index, processed_input)
  File "/home/user/.local/lib/python3.8/site-packages/gradio/blocks.py", line 452, in call_function
    prediction = await anyio.to_thread.run_sync(
  File "/home/user/.local/lib/python3.8/site-packages/anyio/to_thread.py", line 31, in run_sync
    return await get_asynclib().run_sync_in_worker_thread(
  File "/home/user/.local/lib/python3.8/site-packages/anyio/_backends/_asyncio.py", line 937, in run_sync_in_worker_thread
    return await future
  File "/home/user/.local/lib/python3.8/site-packages/anyio/_backends/_asyncio.py", line 867, in run
    result = context.run(func, *args)
  File "/home/user/.local/lib/python3.8/site-packages/gradio/interface.py", line 495, in <lambda>
    lambda *args: self.run_prediction(args)[0]
  File "/home/user/.local/lib/python3.8/site-packages/gradio/interface.py", line 678, in run_prediction
    prediction = predict_fn(*processed_input)
  File "app.py", line 17, in infer
    inputs = load_images([image])
  File "/home/user/app/utils.py", line 21, in load_images
    x = np.clip(file.reshape(480, 640, 3) / 255, 0, 1)
AttributeError: 'NoneType' object has no attribute 'reshape'
```
```


### System Info

```shell
Spaces, Gradio 3.0.13
```

### Severity

serious, but I can work around it

**Comment by abidlabs** [comment]
Thanks @osanseviero, confirming that I see this issue. We'll look into this to see what's going on

**Comment by freddyaboulton** [comment]
I no longer get that same error as the code in the space (as well as the gradio version) have been updated since this was posted.

I'm pretty sure this is the same bug as #1867 as the output is a Gallery and cache_examples is set to true in spaces.

Will fix #1867 to see if this bug is fixed.
